src/avfoundation.cs

```diff
@@ -10317,11 +10335,9 @@ interface AVPlayer {
 		[Notification]
 		NSString AvailableHdrModesDidChangeNotification { get; }
 
-		// From AVPlayer (AVPlayerVideoDecoderGPUSupport) Category
-
-		[NoWatch, NoTV, NoiOS, Mac (10,13,4)]
+		[NoWatch, NoTV, NoiOS, Mac (10, 13)]
```

Comment:
```
[FAIL] Selector not found for AVFoundation.AVPlayer : preferredVideoDecoderGPURegistryID 
[FAIL] Selector not found for AVFoundation.AVPlayer : setPreferredVideoDecoderGPURegistryID: 
```
That's likely incorrect because the tests don't like it...

Reply:
reverting